decidim-core/app/packs/stylesheets/decidim/_dropdown.scss

```
@@ -33,6 +33,23 @@
   }
 }
 
+[data-target="dropdown-menu-main-mobile"] {
+  &[aria-expanded="true"] > span {
+    @apply hidden;
+  }
+
+  &[aria-expanded="false"] > span {
+    @apply block;
+  }
+}
+
+[data-target="dropdown-menu-main-desktop"] {
+  > span,
+  > svg {
+    @apply hidden md:block #{!important};
```

I don't think this important is doing anything, and I'd prefer to try alternatives if this isn't the case. 

```suggestion
    @apply hidden md:block;
```